profiles/kentik_snmp/cisco/cisco-wlc.yml

```diff
@@ -182,6 +182,17 @@ metrics:
           name: cLWlanSsid
           OID: 1.3.6.1.4.1.9.9.512.1.1.1.1.4
         tag: wlan_ssid
+      # This object represents average value of peer controller reachable latency in micro seconds
+      - column:
+          name:  cLHaAvgPeerReachLatency
+          OID: 1.3.6.1.4.1.9.9.843.1.5.1
+        tag: Ha_avg_peer_reach_latency
+      #  This object represents current status of bulk sync
+      - column:
+          name:  cLHaBulkSyncStatus
+          OID: 1.3.6.1.4.1.9.9.843.1.5.3
+        tag:  Ha_bulk_sync_status
```

Comment:
this OID returns a status as a [string value](https://github.com/netdisco/netdisco-mibs/blob/0e65e5f33694d7cfa818b8c26e100fb63bb2e7ae/cisco/CISCO-LWAPP-HA-MIB.my#L985); you'll need to set this up as a standard scalar OID and use the [to_one](https://github.com/kentik/snmp-profiles/blob/7694cd65a24e8ae35ad334209d33e041adf7731e/profiles/kentik_snmp/_template.yml#L98) conversion to force a numeric dimensional metric that will have the `Ha_bulk_sync_status` decoration as the original string value.